.eslintrc.json

```diff
@@ -3,15 +3,20 @@
 		"browser": true,
 		"es2021": true
```

Comment:
I think you should have `"jest/globals": true` in the env entry but no worries since it's working